README.md

```diff
@@ -4,26 +4,53 @@
 [![license](https://img.shields.io/github/license/taylorhakes/python-redis-cache.svg)](https://github.com/taylorhakes/python-redis-cache/blob/master/LICENSE)
 
 # python-redis-cache
+
 Simple redis cache for Python functions
 
 ### Requirements
+
 - Redis 5+
 - Python 3.8+ (should work in Python 3.6+, but not tested)
 
 ## How to install
+
 ```
 pip install python-redis-cache
 ```
 
 ## How to use
+
+The Redis cache client can either connect to a standalone Redis server or to a sentinel managed Redis setup. Here's how you can initialize the cache client for these scenarios:
+
+### Standalone Redis setup
+
 ```python
 from redis import StrictRedis
 from redis_cache import RedisCache
 
 client = StrictRedis(host="redis", decode_responses=True)
 cache = RedisCache(redis_client=client)
+```
 
+### Sentinel Managed Redis setup
 
+```python
+from redis.sentinel import Sentinel
+from redis_cache import RedisCache
+
+# Here, redis-sentinel1 is the sentinel service name and mymaster is the name
+# of the master service.
+sentinel = Sentinel([('redis-sentinel1', 26379)],
+    password='topsecret',                               # redis server password
+    sentinel_kwargs={"password": "sentinel-topsecret"}  # sentinel password
+)
+
+cache = RedisCache(redis_client=sentinel)
```

Comment:
@glesperance Thanks for submitting this pull request. Can you use Sentinel with the current library with the following code? No changes needed to the library?
```python
sentinel = Sentinel([('redis-sentinel1', 26379)],
    password='topsecret',                               # redis server password
    sentinel_kwargs={"password": "sentinel-topsecret"}  # sentinel password
)

cache = RedisCache(redis_client=sentinel.master_for(master_name))
```

Are the changes to the library needed or am I missing something?

Reply:
We can but this will break as soon as the redis master changes. Getting the proper client at runtime is important for the high availability scenario I'm working with

Reply:
That makes sense. Can you describe the situation in more detail? 

I was taking a look that the code in redis-py and it seems when someone calls `master_for` it creates a connection pool for the `service_name` passed into `master_for`. If it runs into a connection error, it seems to pull the new master (it doesn't hard code the ip connection). Maybe I am way off. Are you seeing something different?
https://github.com/redis/redis-py/blob/master/redis/sentinel.py#L324

Reply:
@taylorhakes perhaps I am reading this wrong but I don't see that in the code you linked. Do you have a link where it catches the connection error and reconnects to the master?

To be clear, that would make the most sense from redis-py POV

Reply:
I believe it's here? Sorry I didn't link to the exact logic on connect.

https://github.com/redis/redis-py/blob/master/redis/sentinel.py#L45

Reply:
Here is the logic to retry https://github.com/redis/redis-py/blob/master/redis/sentinel.py#L71

Reply:
Gotcha. Thanks @taylorhakes !